src/ControllerFactory.php

```diff
@@ -2,7 +2,7 @@
 
 namespace Laminas\ApiTools\Documentation;
 
-use Interop\Container\ContainerInterface;
+use interop\container\containerinterface;
```

Comment:
phpcs seems somehow make it lowercase ?

Reply:
This is due to moving to laminas-servicemanager 3.11; I'll fix this now.

Reply:
Fixed (there were two locations for this)